frontend/src/components/phones/dashboard/SendersPanelView.tsx

```diff
@@ -0,0 +1,186 @@
+            >
+              {data.last_inbound_type === "text" && (
+                <ForwardedTextIcon
+                  alt="Last received a text"
+                  className={styles["forwarded-type-icon"]}
+                  width={20}
+                  height={12}
+                />
+              )}
+              {data.last_inbound_type === "call" && (
+                <ForwardedCallIcon
+                  alt="Last received a call"
+                  className={styles["forwarded-type-icon"]}
+                  width={20}
+                  height={15}
+                />
+              )}
+              <Moment calendar={calendarStrings}>
+                {data.last_inbound_date}
+              </Moment>
+            </span>
+            <span className={styles["sender-controls"]}>
+              <button
+                onClick={() =>
+                  inboundContactData.setForwardingState(!data.blocked, data.id)
+                }
+                className={styles["block-btn"]}
+              >
+                {data.blocked ? "Unblock" : "Block"}
+              </button>
+            </span>
+          </li>
+        );
+      });
+
+  const senderLogsPanel = (
+    <ul className={styles["caller-sms-senders-table"]}>
+      <li className={styles["greyed-contact"]}>
+        <span>
+          {l10n.getString("phone-dashboard-sender-table-title-sender")}
+        </span>
+        <span>
+          {l10n.getString("phone-dashboard-sender-table-title-activity")}
+        </span>
+        <span>
+          {l10n.getString("phone-dashboard-sender-table-title-action")}
+        </span>
+      </li>
+      {inboundContactArray}
+    </ul>
+  );
+
+  return (
+    <div id="secondary-panel" className={styles["dashboard-card"]}>
+      <div className={styles["dashboard-card-caller-sms-senders-header"]}>
+        <span>
+          <button
+            type="button"
+            onClick={() => props.back_btn()}
+            className={styles["caller-sms-logs-back-btn"]}
+          >
+            <ChevronLeftIcon
+              alt="Back to Primary Dashboard"
+              className={styles["nav-icon"]}
+              width={20}
+              height={20}
+            />
+          </button>
+        </span>
+        <span className={styles["caller-sms-logs-title"]}>
+          {l10n.getString("phone-dashboard-senders-header")}
+        </span>
+        <span></span>
+      </div>
+      {props.type === "primary" && <>{senderLogsPanel}</>}
+      {props.type === "disabled" && <>{disabledCallerSMSSendersPanel}</>}
+      {props.type === "empty" && <>{emptyCallerSMSSendersPanel}</>}
+    </div>
+  );
+};
+
+function formatPhoneNumberToUSDisplay(e164Number: string) {
+  const friendlyPhoneNumber = e164Number.split("");
+  friendlyPhoneNumber?.splice(2, 0, " (");
+  friendlyPhoneNumber?.splice(6, 0, ") ");
+  friendlyPhoneNumber?.join("");
+  return friendlyPhoneNumber;
+}
```

Comment:
question (blocking): Should we remove this @groovecoder since we now have the library to handle phone numbers and we're using it on the dashboard already? I think we already merged it - yea, #2357 introduces the adoption of the `react-phone-number-input` package.

Reply:
Yeah, let's consistently use the same library to format phone numbers.